freqtrade/optimize/hyperopt_tools.py

```diff
@@ -233,18 +233,19 @@ def get_result_table(config: dict, results: list, total_epochs: int, highlight_b
                              'results_metrics.winsdrawslosses',
                              'results_metrics.profit_mean', 'results_metrics.profit_total_abs',
                              'results_metrics.profit_total', 'results_metrics.holding_avg',
+                             'results_metrics.max_drawdown', 'results_metrics.max_drawdown_abs',
                              'loss', 'is_initial_point', 'is_best']]
         else:
             # Legacy mode
             trials = trials[['Best', 'current_epoch', 'results_metrics.trade_count',
                              'results_metrics.winsdrawslosses',
                              'results_metrics.avg_profit', 'results_metrics.total_profit',
-                             'results_metrics.profit', 'results_metrics.duration',
-                             'loss', 'is_initial_point', 'is_best']]
+                             'results_metrics.profit', 'results_metrics.duration', 'results_metrics.max_drawdown',
+                             'results_metrics.max_drawdown_abs', 'loss', 'is_initial_point', 'is_best']]
```

Comment:
without even looking at what ci sais (i know it'll fail) - this will kill you as these fields are not available (and can't be).

"legacy" mode is for when the backtest-results were not stored (it's used by show_hyperopt / list-hyperopt commands if using a pickle file) - so this data is not available as it hasn't been stored (which only changed a version or 2 ago).

I suspect you'll need to make formatting of this optional (not displaying the column if we're in "legacy mode" - otherwise it'll display an empty column (as you'll need to add that column first) - which won't be much better either.

Reply:
Thanks for the clarification on legacy mode. I was a little doubtful about which path to follow for correct implementation, I thought that the code would be better duplicating the definition of the columns as a whole to avoid later manipulation of the dataframe columns.

Reply:
The second option would be to temporarily create the columns and then remove them (as in winsdrawslosses column).

Reply:
I ended up choosing the same implementation as the winsdrawslosses column, even though it only looks for drawdown if it is not legacy mode, the unitary test failed.

Reply:
Looking at the legacy mode was not enough to guarantee the existence of the column in the result, I had to create a control variable. If you think there is a more "correct" way to do this, let me know.